boomerang.js

```diff
@@ -1,44 +1,56 @@
-/*
- * Copyright (c) 2011, Yahoo! Inc.  All rights reserved.
- * Copyright (c) 2012, Log-Normal, Inc.  All rights reserved.
- * Copyright (c) 2014, SOASTA, Inc. All rights reserved.
+/**
+ * @copyright (c) 2011, Yahoo! Inc.  All rights reserved.
+ * @copyright (c) 2012, Log-Normal, Inc.  All rights reserved.
+ * @copyright (c) 2012-2016, SOASTA, Inc. All rights reserved.
  * Copyrights licensed under the BSD License. See the accompanying LICENSE.txt file for terms.
  */
 
 /**
-\file boomerang.js
-boomerang measures various performance characteristics of your user's browsing
-experience and beacons it back to your server.
-
-\details
-To use this you'll need a web site, lots of users and the ability to do
-something with the data you collect.  How you collect the data is up to
-you, but we have a few ideas.
+ * @namespace Boomerang
+ * @desc
+ * boomerang measures various performance characteristics of your user's browsing
+ * experience and beacons it back to your server.
+ *
+ * To use this you'll need a web site, lots of users and the ability to do
+ * something with the data you collect.  How you collect the data is up to
+ * you, but we have a few ideas.
 */
 
-// Measure the time the script started
-// This has to be global so that we don't wait for the entire
-// BOOMR function to download and execute before measuring the
-// time.  We also declare it without `var` so that we can later
-// `delete` it.  This is the only way that works on Internet Explorer
+/**
+ * @memberof Boomerang
+ * @type {TimeStamp}
+ * @desc
+ * Measure the time the script started
+ * This has to be global so that we don't wait for the entire
+ * BOOMR function to download and execute before measuring the
+ * time.  We also declare it without `var` so that we can later
+ * `delete` it.  This is the only way that works on Internet Explorer
+*/
 BOOMR_start = new Date().getTime();
 
 /**
- Check the value of document.domain and fix it if incorrect.
- This function is run at the top of boomerang, and then whenever
- init() is called.  If boomerang is running within an iframe, this
- function checks to see if it can access elements in the parent
- iframe.  If not, it will fudge around with document.domain until
- it finds a value that works.
-
- This allows customers to change the value of document.domain at
- any point within their page's load process, and we will adapt to
- it.
+ * @function
+ * @desc
+ * Check the value of document.domain and fix it if incorrect.
+ * This function is run at the top of boomerang, and then whenever
+ * init() is called.  If boomerang is running within an iframe, this
+ * function checks to see if it can access elements in the parent
+ * iframe.  If not, it will fudge around with document.domain until
+ * it finds a value that works.
+ *
+ * This allows customers to change the value of document.domain at
+ * any point within their page's load process, and we will adapt to
```

Comment:
Should this paragraph be here/refer to `customers`? Does that make sense for OSS users?